src/client/components/MainNavBar/ProfileName.tsx

```diff
@@ -166,17 +177,15 @@ export function ProfileName({
   }, [person, company, history, profileAttribute, profileType]);
 
   return (
-    <span
-      className={classnames(
-        styles.ProfileName,
-        styles[`ProfileName--${profileType}`]
-      )}
-    >
-      {isAppStateReady ? (
-        nameContent
+    <>
+      {appStateFirstLoad === true ? (
```

Comment:
Zou je hier niet de variabele `isAppStateReady` kunnen controleren?

Reply:
Kan wel maar geeft een lelijk en onnodig effect. Klikken op de toggle zorgt ervoor dat AppState wordt ge-update en daardoor niet ready is. Hierdoor zien we een lader, smaller dan naam + bedrijfsnaam, heel snel hierna is de appState wél ready. Dit geeft een soort flicker-effect. Dat is niet mooi.